main.js

```diff
@@ -0,0 +1,78 @@
+// constant variable declare
+
+const addButton = document.getElementById('Add');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const adder = document.getElementById('keeper');
+
+// setup the local storage
+
+if (localStorage.getItem('books') !== null) {
+  const getbook = JSON.parse(localStorage.getItem('books'));
+
+  getbook.forEach((item) => {
+    adder.innerHTML += `
+        <div>
+          <p>${item.title}</p>
+          <p>${item.author}</p>
+          <button class="remove" name="${item.title}">Remove</button>
+          <hr>
+        </div>
+      `;
+  });
+}
+
+// function for Book info
+
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+// function to add which book detial on page
+
+addButton.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  // array usage
+
+  adder.innerHTML += `
+    <div> 
+    <p>${title.value}</p>
+    <p>${author.value}</P>
+    <button class ="remove" name="${title.value}">Remove book </button> 
+    <hr>
+    </div>
+    `;
+
+  // adding book with local storage
```

Comment:
- [ ] Please create a code to add data to the collection that keeps a list of books. It's a project requirement.